src/libraries/System.Net.Security/src/System/Net/Security/SslStreamCertificateContext.Linux.cs

```diff
@@ -7,6 +7,15 @@ namespace System.Net.Security
 {
     public partial class SslStreamCertificateContext
     {
+        // No leaf, no root.
+        private const int _trimChain = 2;
```

Comment:
Style: Consts should be PascalCase.

Since it's always "no leaf", maybe it should just be

```C#
private const bool TrimRoot = true;
```

(and false for Windows)

Reply:
I was thinking about it but in the places I needed (e.g. loop end condition) int was more convenient. 
TrimRoot sounds better. I will take a look.